src/page-objects/full-page-screenshot.ts

```diff
@@ -51,9 +51,9 @@ export async function scrollAndMergeStrategy(browser: WebdriverIO.Browser) {
   return mergeImages(screenshots, width * pixelRatio, height * pixelRatio, lastImageOffset * pixelRatio, offsetTop);
 }
 
-export default async function fullPageScreenshot(browser: WebdriverIO.Browser) {
+export default async function fullPageScreenshot(browser: WebdriverIO.Browser, forceScrollAndMerge: boolean = false) {
```

Comment:
It's a nitpick, but from my perspective, `forceScrollAndMerge` doesn't really reflect what this variable does. Maybe `skipPuppeteer` would be better?

Reply:
@georgylobko I understand your perspective. There are two screenshot strategies (Puppeteer and ScrollAndMerge) and we default to Puppeteer assuming it exists. The semantics of forcing ScrollAndMerge sounds more accurate to me because we are explicitly saying we want to utilize that strategy regardless of the Puppeteer status.

Reply:
I also have a partner CR for the integration tests package that uses the same semantics and I'd prefer not to update it in both places. :)

Reply:
Got it! Thanks for the additional context